packages/astro/README.md

```diff
@@ -0,0 +1,34 @@
+# Trigger.dev native integration add-on for Astro
+
+This Astro integration project provides a `createAstroRoute` function that can be used to create an API endpoint route in your Astro project for integration with the hosted Trigger.dev platform.
+
+## Usage
+
+1. Install the package:
+
+```bash
+npm install --save https://github.com/lirantal/triggerdev-astro-integration
```

Comment:
These instructions should be updated to reference the final package `@trigger.dev/astro`

Reply:
Fixed